users/assets/login_authenticate.py

```diff
@@ -0,0 +1,13 @@
+from django.shortcuts import redirect
+from django.contrib.auth import authenticate, login
+
+
+def login_authenticate(request, form_data):
+    print(form_data)
```

Comment:
@ahmadx16 Print statements shouldn't be part of production code, instead, use logging.